### ISSUE TITLE
Update dbquery docs

### ISSUE THREAD

**Issue by dkirkby** [DarkEnergyScienceCollaboration/WeakLensingDeblending#7]
PR #5 replaces mssql with sqlalchemy, so this changes the instructions for running dbquery.  Note that sqlalchemy is included with anaconda.

**Comment by dkirkby** [comment]
@alexabate I just noticed that you used `host="localhost"` when you updated `dbquery` to use sqlalchemy (see [here](https://github.com/DarkEnergyScienceCollaboration/WeakLensingDeblending/blob/master/dbquery.py#L95)) Is that because you were already logged on to the server at UW?

**Comment by alexabate** [comment]
Yes I believe so. I think I logged in to UW in another terminal window
